📜 Initialize splittable checkout rest api order custom references connector bundle

daniel-rose opened the pull request:
- add plugin to expand quote with order custom references
- prepare transfer definition
- ...

daniel-rose commented:
@DerCalli please use POST for resource "return-labels". https://restfulapi.net/http-methods/

daniel-rose reviewed (changes requested):
@DerCalli please use POST for resource "return-labels". https://restfulapi.net/http-methods/

DerCalli commented:
> @DerCalli please use POST for resource "return-labels". https://restfulapi.net/http-methods/

@daniel-rose: i already talked about that with chris yesterday. it was a wish from frontend to use GET instead POST. i was a little bit confused about that. i will turn in back

daniel-rose committed:
📜 Initialize splittable checkout rest api order custom references connector bundle
- add plugin to expand quote with order custom references
- prepare transfer definition
- ...

daniel-rose committed:
KIS
